### ISSUE TITLE
[QUESTION] Are there still known issues between WingetUI and Scoop?

### ISSUE THREAD

**Issue by ppvnf** [martinet101/WingetUI#310]
There's this warning in the readme:

⚠️ Currently there are known issues between WingetUI and Scoop!
Please install the app with its own installer or through Winget for the time being. ⚠️

Is it still true? I've installed WingetUI through Scoop and couldn't find any problem with the latest version

**Comment by martinet101** [comment]
Yes, i think it was related to scoop issues but they have been fixed now. gona remove that
